docs/setup-cluster/checklists/_index.rst

```diff
@@ -321,6 +321,16 @@ users. To find out more, visit :ref:`run-as-user` and :ref:`run-unprivileged-tas
 
    Red Hat OpenShift users should not configure non-root containers.
 
+To run containers as non-root users:
+
+-  Create a non-root user in Dockerfile.
```

Comment:
You should not have to touch the dockerfile for our non-root container support to work

Reply:
i'll remove that bullet point

Reply:
My attempt at a test plan for a checklist:
```
- Choose a user who should run as non-root, or create a temporary one.

- Link the user to the actual username/UID and groupname/GID that users' tasks should run as:

  det user link-with-agent-user \
    --agent-user $THE_USER \
    --agent-uid  $THE_UID \
    --agent-group $THE_GROUP \
    --agent-gid $THE_GID \
    $THE_DETERMINED_USER

- Start a shell as the specified user:

    det -u $THE_DETERMINED_USER shell start

- Make sure that you have the expected username/UID and groupname/GID by executing `id -a` inside the shell
   
- Now you know that nonroot containers are working; you should also run your normal training jobs as the modified determined user, to make sure your applications work without root.
```